spec/fixtures/files/publicart_movement_row11.xml

```diff
@@ -14,6 +14,7 @@
     <inventoryContactList/>
     <movementMethods/>
     <currentLocation>urn:cspace:publicart.collectionspace.org:locationauthorities:name(location):item:name(SampleLocationText91588831925061)'Sample Location Text 9'</currentLocation>
+    <movementContact>urn:cspace:publicart.collectionspace.org:personauthorities:name(person_shared):item:name(SamplePerson11589796879672)'Sample Person 1'</movementContact>
```

Comment:
Added (copied XML element from row3.xml) because the test expected this not to be empty and I didn't want to change the test pattern!